### ISSUE TITLE
Error C3-14978-9 

### ISSUE THREAD

**Issue by Ryogo-Z** [codestation/qcma#37]
This error will be thrown after psp2-updatelist.xml request by the Vita if psp2-updatelist.xml returned to the Vita wouldn't contain FW info for the request region.
Current integrated in QCMA psp2-updatelist.xml contains next regions: au, eu, jp, kr, mx, ru, tw, uk, us.
In my case console was requesting FW update info for the "sa" region which is missing from the current integrated psp2-updatelist.xml.

**Comment by Ryogo-Z** [comment]
BTW, I've got a suggestion. 

psp2-updatelist.xml url looks like: http://fxx01.psp2.update.playstation.net/update/psp2/list/jp/psp2-updatelist.xml - where "xx" is region id;

So, maybe instead currently integrated psp2-updatelist.xml file QCMA should integrate next template:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<update_data_list>
<region id="REG_ID">
    <np level0_system_version="00.000.000" level1_system_version="00.000.000" level2_system_version="00.000.000" map="00.000.000" />
    <np_d level0_system_version="00.000.000" level1_system_version="00.000.000" level2_system_version="00.000.000" map="00.000.000" />
    <version system_version="00.000.000" label="0.00">
      <update_data update_type="full">
        <image size="0">http://www.example.com/PSP2UPDAT.PUP</image>
      </update_data>
    </version>
</region>
</update_data_list>
```
And replace "REG_ID" in template with region id parsed from request url?

**Comment by codestation** [comment]
Seems like fxx01 doesn't work for all regions, for example CMA isn't gonna make a request to fmx01.* since that server doesn't have a update xml file.

Probably taking the region from the XX  in http://fjp01.psp2.update.playstation.net/update/psp2/list/XX/psp2-updatelist.xml is a better choice but i don't have time right now to test it.

Gonna settle into adding the extra region to the XML for now, if this breaks again then i could add extra code to parse the url and replace the region in the xml file.

Fixed (for now) in b5e4462db992f956e471143f704909c8f7bdee27
